contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -135,10 +135,13 @@ BEGIN
 
   CREATE OR REPLACE VIEW msdb_dbo.syspolicy_configuration
   AS
-    SELECT
-      CAST(NULL AS sys.SYSNAME) AS name,
-      CAST(NULL AS sys.sql_variant) AS current_value
-    WHERE FALSE; -- Condition will result in view with an empty result set
+    SELECT CAST(t.name AS sys.sysname), CAST(t.current_value AS sys.sql_variant) FROM
```

Comment:
Just make sure that base type is correct for t.current_values.

Reply:
Good point. I tested in SQL Server what the base type should be for `current_value` using this query:
```
DECLARE @MyVariant sql_variant = (select current_value from msdb.dbo.syspolicy_configuration where name='Enabled')
SELECT sql_variant_property(@MyVariant ,'basetype');
```

And it reports the base type as being `int` for these 3 `name` values. 

I tested the base type using the same query on BBF using my implementation, and it also correctly reports the base type as being `int`. Just to be safe I'll explicitly cast `t.current_value` to `int` anyway. Done.